tools/build_defs/repo/utils.bzl

```diff
@@ -385,3 +385,18 @@ def use_netrc(netrc, urls, patterns):
             }
 
     return auth
+
+def read_user_netrc(ctx):
+    "Read user's default netrc file"
```

Comment:
Argument "ctx" is not documented; return value of "read_user_netrc" is not documented.

If the documentation for the argument exists but is not recognized by Buildifier
make sure it follows the line "Args:" which has the same indentation as the opening """,
and the argument description starts with "<argument_name>:" and indented with at least
one (preferably two) space more than "Args:", for example:

    def read_user_netrc(ctx):
        """Function description.

        Args:
          ctx: argument description, can be
            multiline with additional indentation.
        """